install/installer/cloud_cli_setup

```diff
@@ -27,7 +27,8 @@ TRAF_CONFIG=/etc/trafodion/trafodion_config
 source $TRAF_CONFIG
 
 if [ $node_count -eq 1 ]; then
-    exit 0
```

Comment:
Cloud cli set up scripts needs to run on clusters  where node count  is > 1 . So we need to just exit here